libraries/classes/Server/Select.php

```diff
@@ -98,39 +81,45 @@ public static function render($not_only_options, $omit_fieldset)
             }
 
             if ($list) {
-                $retval .= '<li>';
                 if ($selected) {
-                    $retval .= '<strong>' . htmlspecialchars($label) . '</strong>';
+                    $servers['list'][] = [
+                        'selected' => true,
+                        'label' => $label,
+                    ];
                 } else {
                     $scriptName = Util::getScriptNameForOption(
                         $GLOBALS['cfg']['DefaultTabServer'],
                         'server'
                     );
-                    $retval .= '<a class="disableAjax item" href="'
-                        . $scriptName
-                        . Url::getCommon(['server' => $key], strpos($scriptName, '?') === false ? '?' : '&')
-                        . '" >' . htmlspecialchars($label) . '</a>';
+                    $href = $scriptName . Url::getCommon(
+                        ['server' => $key],
+                        strpos($scriptName, '?') === false ? '?' : '&'
+                    );
+                    $servers['list'][] = [
+                        'href' => $href,
+                        'label' => $label,
+                    ];
                 }
-
-                $retval .= '</li>';
             } else {
-                $retval .= '<option value="' . $key . '" '
-                    . ($selected ? ' selected="selected"' : '') . '>'
-                    . htmlspecialchars($label) . '</option>' . "\n";
+                $servers['select'][] = [
+                    'value' => $key,
+                    'selected' => $selected ? $selected : '',
```

Comment:
```suggestion
                    'selected' => $selected ? 'true' : '',
```
Would it be more clear ?